test/fixtures/tags-config.yml

```diff
@@ -0,0 +1,3 @@
+tag_protection:
```

Comment:
lets simplify this to `tags` to be more similar to the existing `branches`, for example